added text descritpion and changed top padding

benpepin opened the pull request.

benpepin committed:
added text descritpion and changed top padding

benpepin committed:
responsiveness for mobil

strickinato committed:
kill extraneous lines and spaces

strickinato commented on source/index.html.erb:
```diff
@@ -4,6 +4,7 @@ title: browserstats.love
 <section class="top-bar">
   <h2 class="site-title"><span id="title-emoji">❤️</span>    browserstats.love</h2>
+  <h2 class="site-description">A tropical site tracking broswer popularity</h2>
   <h2 class="site-updated">Last Updated <%= browser_data["last_update"] %></h2>
```
You have a typo- it's `browser` not `broswer`

strickinato commented on source/javascripts/site.js:
```diff
@@ -1,6 +1,7 @@
 
 
+
 $(function(){
```
no reason to check in extraneous newline

strickinato commented on source/stylesheets/site.css:
```diff
@@ -18,6 +18,54 @@ a {
 
 
+.top-bar {
+
```
whitespace

strickinato commented on source/stylesheets/site.css:
```diff
@@ -18,6 +18,54 @@ a {
+
+.mobile {
+  display:none;
+}
```
there's nothing with the class mobile - so these don't do anything

strickinato committed:
kill code that isn't actually doing anything

you're selecting for
.mobile (but nothing in html has that class)
.desktop (but nothing in html has that class either)

strickinato committed:
move the top bar code so it's all together

strickinato reviewed (commented):
Looks good Ben -- you have some code that wasn't actually doing anything... in particular, the `.mobile` and `.desktop` classes were in the css but not in the HTML

I've made some commits you should look at
